gem/lib/illuminator/xcode-builder.rb

```diff
@@ -84,7 +84,9 @@ def _buildCommand
       command << 'xcodebuild'
       command << parameters << environmentVars << tasks
       command << " | tee '#{self.logfilePath}'"
-      command << " | xcpretty -c -r junit" unless Illuminator::HostUtils.which("xcpretty").nil?  # use xcpretty if available
+      unless Illuminator::HostUtils.which("xcpretty").nil?  # use xcpretty if available
+        command << " | xcpretty -c -r junit -o #{BuildArtifacts.instance.xcprettyReportFile}"
```

Comment:
it seems to put output in `build/reports/junit.xml` instead of the specified `buildArtifacts` directory